01/app.js

```diff
@@ -1 +1,14 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const comments = document.querySelectorAll(
+    '.comments__item, .comments__item-newest'
+);
```

Comment:
Czy, aby na pewno to prawidłowe rozwiązanie? Szukamy elementu, który ma obie klasy, a nie jedną z nich :)